index.html

```diff
@@ -2,10 +2,66 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="author" content="Juan Nunez" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Bio Page</title>
+    <!--Font Links Section-->
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <!--CSS Links Section-->
+    <link rel="stylesheet" href="index.css">
+
+    <title>Juan Nunez Bio</title>
 </head>
 <body>
+    <header>
+        <nav class="topnav">
+            <ul class="navul">             
+                <li class="navli"><a href="index.html">Home</a></li>|
```

Comment:
Avoid including an active link to the current page (i.e. a link to 'Home' on the 'Home' page is unnecessary -- if clicked, it forces to page to reload from the Web Server).  Consider replacing the 'Home' link with the text "Home".  This does require a specific tag (a 'p' tag will force an end-of-paragraph to occur, pushing the other header elements down a line), but it is worth knowing how to achieve that state.